File: 200_code_python/9.py
# Viết bằng PYTHON
# Tính diện tích hình tam giác

# Nhập chiều cao và đáy từ bàn phím
h = float(input("Nhap chieu cao h cua hinh chu nhat: "))
d = float(input("Nhap do dai day d cua hinh chu nhat: "))

# Kiểm tra điều kiện chiều cao và độ dày đáy > 0
if h > 0 and d > 0 :
    # Tính diện tích hình chữ nhật
    S = 0.5 * h * d
    # In kết quả
    print("Dien tich hinh chu nhat la: ", S)
else: 
    print("Chieu cao và do dai day phai lon hon 0.")

File: 200_code_cpp/9.cpp
// Viết bằng C++
// Tính diện tích hình tam giác

#include <iostream>
using namespace std;

int main() {
    // Khai báo biến
    float h, d;

    // Nhập chiều cao và đáy từ bàn phím
    cout << "Nhap chieu cao h cua hinh tam giac: ";
    cin >> h;
    cout << "Nhap do dai d cua hinh tam giac: ";
    cin >> d;
    
    // Kiểm tra điều kiện chiều cao và đáy > 0
    if ( h  > 0 && d > 0) {
        // Tính diện tích hình tam giác
        float S = 0.5 * d * h;
        // In kết quả
        cout << "Dien tich hinh tam giac: " << S << endl;
    } else {
        cout << " Chieu cao và day lon hon 0." << endl;
    }

    return 0;
}
